src/index.css

```diff
@@ -1,10 +1,15 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #a7d4ff;
 }
 
 code {
```

Comment:
let's find proper reset.scss or normalize.scss. hint, using all selectors standalone is faster than * - you will find a lot of ready resets/normalizes on the internet. Find the one that will fit your styling style. You have to be careful though, because you may encounter problems that will be done by those reset/normalize features